Assignment2.java

```diff
@@ -0,0 +1,125 @@
+
+package assignment2;
+import java.util.Scanner;
+
+/**
+ *
+ * @author Samantha
+ */
+public class Assignment2 {
+    
+    public static void main(String[] args) {
+        
+       Boolean bool = true;   // Determines whether user will contine application
+       Scanner keyboard = new Scanner(System.in);
+       Integer input; //User menu selection
+       Integer i;
+       String[] menuItems = {
+           "Word Value Calculator", "Grade Calculator","Song Lyrics Loop",
+        "...Under Construction...","...Under Construction...", "Exit"
+       };
+       
+       // DISPLAY MENU
+       for(i=0; i < menuItems.length; i++) {
+           System.out.println((i+ 1) + ". " + menuItems[i]);
+       }
+       
+       
+        while (bool == true) {
```

Comment:
I agree with @ztmurphy21 that I appreciate you using next weeks material. It helps me gauge acceptance of the future material and helps me plan how much time we will spend on it.